.pre-commit-config.yaml

```diff
@@ -0,0 +1,29 @@
+repos:
+  - repo: local
+    hooks:
+      - id: shfmt
+        name: shfmt
+        minimum_pre_commit_version: 2.4.0
+        language: golang
+        additional_dependencies: [mvdan.cc/sh/v3/cmd/shfmt@v3.2.2]
+        entry: shfmt
+        args: [-i=2, -ci, -w]
+        types: [shell]
+      - id: shellcheck
```

Comment:
Maybe some newlines between each id to mark a clear separation :) ?

Reply:
It's now been formatted according to `prettier` :)
We can iterate on this file in the feature once everybody has used it a bit more.